sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/AbstractReactiveDatabaseIT.java

```diff
@@ -0,0 +1,243 @@
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public abstract class AbstractReactiveDatabaseIT {
+    @Test
+    public void getAll() {
+        getApp().given()
+                .get("/library/books")
+                .then()
+                .statusCode(HttpStatus.SC_OK);
+    }
+
+    @Test
+    public void connectToUniEndpoint() {
+        Response response = given()
+                .when().get("/library/books/1");
+        String title = response.body().asString();
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Assertions.assertEquals("Slovník", title);
+    }
+
+    @Test
+    public void findById() {
+        String title = given()
+                .when().get("/library/books/2")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("Thinking fast and slow", title);
+    }
+
+    @Test
+    public void findByInvalidId() {
+        Response response = given().when().get("/library/books/256");
+        Assertions.assertEquals(HttpStatus.SC_NOT_FOUND, response.statusCode());
+    }
+
+    @Test
+    public void connectToMultiEndpoint() {
+        String result = given()
+                .when().get("/library/books")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertTrue(result.contains("Slovník"));
+        Assertions.assertTrue(result.contains("Thinking fast and slow"));
+    }
+
+    @Test
+    public void searchByQuery() {
+        given()
+                .when().get("/library/authors")
+                .then()
+                .statusCode(HttpStatus.SC_OK);
+    }
+
+    @Test
+    public void searchWithJoin() {
+        String result = given()
+                .when().get("/library/books/author/Kahneman")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertTrue(result.contains("Attention and Effort"));
+        Assertions.assertTrue(result.contains("Thinking fast and slow"));
+    }
+
+    @Test
+    public void searchWithLimit() {
+        String result = given()
+                .when().get("/hibernate/books/author/4")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertFalse(result.contains("Attention and Effort"));
+        Assertions.assertTrue(result.contains("Thinking fast and slow"));
+    }
+
+    @Test
+    public void namedQuery() {
+        Response response = given()
+                .when().get("/hibernate/books/starts_with/Thinking");
+        String result = response.getBody().asString();
+        Assertions.assertTrue(result.contains("Thinking fast and slow"));
+    }
+
+    @Test
+    @Order(3)
```

Comment:
Put orders on top of the class + sort them

Reply:
done